src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -215,8 +216,12 @@ public function getContentData(PropertyInterface $property)
         $filters = $property->getValue();
         $filters['excluded'] = [$property->getStructure()->getUuid()];
 
+        // extends selected filter with requested tags
+        $filters = array_merge(['tags' => []], $filters);
```

Comment:
Still looks a bit strange to me, feels like the JavaScript way, and I didn't see that too often in PHP (probably because `array_merge` hasn't got the best performance).

But what about:

```php
if (array_key_exists('tags', $filters)) {
    $filters['tags'] = [];
}
```

Is it just me, or does it feel cleaner?